sensor/common/sensor/sensor.go

```diff
@@ -327,7 +328,7 @@ func (s *Sensor) newScannerDefinitionsRoute(centralEndpoint string, centralCerti
 	// We rely on central to handle content encoding negotiation.
 	return &routes.CustomRoute{
 		Route:         "/scanner/definitions",
-		Authorizer:    or.Or(idcheck.ScannerOnly(), idcheck.ScannerV4IndexerOnly()),
+		Authorizer:    or.Or(idcheck.ScannerOnly(), idcheck.ScannerV4IndexerOnly(), idcheck.CollectorOnly()),
```

Comment:
Does it mean, that whoever presents the Collector certs will be authorized to call this endpoint?

Reply:
Yes, that is what this means. It's a general problem with cert-based auth that we have in our codebase with all components.
The cert in the secret is the only thing that our components use for authorization.